docs/advanced/naming.md

```diff
@@ -164,16 +202,19 @@ Anime.Name.2022.S01.E01-E11.1080p.BluRay.FLAC2.0.H.265-Group
 Anime.Name.2022.S01.E12-E23.1080p.BluRay.FLAC2.0.H.265-Group
 ```
```

Comment:
Also reduce these to avoid fragmentation, the less variants the better. Emphasize on why it's important to put group at the end for folder names (to allow for alphabetical sorting on user's end)

```
Anime Name (2022) S01 (BD 1080p HEVC Opus) [Dual Audio]-Group
Anime.Name.2022.S01.1080p.AMZN.WEB-DL.DUAL.DDP2.0.H.264-Group
Anime Name - S01E01-E24 - (BD Remux 1080p x264 8-bit DTS-HD) [Dual-Audio]-Group
Anime.Name.2022.S01.1080p.BluRay.REMUX.AVC.FLAC.2.0-Group
```